Makefile

```diff
@@ -99,6 +99,12 @@ dev-quick-install:
 dev-mass-pdf-upload:
 	@docker compose run --rm web python manage.py dev_pdf_mass_upload
 
+dev-mass-pdf-upload:
+	@docker compose run --rm web python manage.py dev_pdf_mass_upload
```

Comment:
I guess this works, but the duplicate is a bit odd. Let's remove it in case some make implementations don't like the second definition of `dev-mass-pdf-upload`